Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -1131,12 +1130,6 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
     @MainActor
     private func handleSettingsChange(_ change: VPNSettings.Change) async throws {
         switch change {
-        case .setExcludeLocalNetworks:
-            if case .connected = connectionStatus {
-                try await updateTunnelConfiguration(
-                    updateMethod: .selectServer(currentServerSelectionMethod),
-                    reassert: false)
-            }
```

Comment:
Not sure if the extension or main app should decide when to restart the adapter.  Maybe I should just do it here?

But doing it here prevents us from doing batch changes - although to be honest we don't do them :)